packages/network-of-terms-catalog/catalog/datasets/genre_filmmuseum.jsonld

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.beeldengeluid.nl/gtaa/Genre_Filmmuseum",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Genre Filmmuseum"

```

Comment:
Should `GTAA` be part of this name? `Genre Filmmuseum` looks really weird. Did you mean: `Genres Filmmuseum`?

Reply:
It’s okay to add non-GTAA datasets, of course. I’m just wondering if this dataset is _only_ being used by Filmmuseum or are other institution also using it or going to use it?

Reply:
I only know of EYE as a user of Genres Filmmuseum.